kubevirt_api/tests/test_e2e.py

```diff
@@ -0,0 +1,25 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+pytestmark = [pytest.mark.e2e]
+
+
+def test_e2e_connect_ok(dd_agent_check):
+    # Since we are using port-forward to query the pods in the kind environment
+    # the check will raise an exception because it cannot find a pod with the ip given in the input
+    with pytest.raises(Exception):
+        aggregator = dd_agent_check()
+        aggregator.assert_metric("kubevirt_api.can_connect", value=1)
```

Comment:
If we're raising an exception I think this assert is not checked

Reply:
Fixed